rpcd/playbooks/verify-maas.yml

```diff
@@ -52,3 +52,6 @@
       delay: "5"
       when:
         - maas_use_api | bool
+
+  vars:
+    maas_use_api: true
```

Comment:
Hmm, I checked and it was [true previously](https://github.com/rcbops/rpc-openstack/pull/2166/files#diff-1d0825392161b9f338fa4db5ddd23713L616). However we do need to be able to override as we use false for PR jobs.